runtime/common/src/weights/xcmp_queue.rs

```diff
@@ -70,4 +70,28 @@ impl<T: frame_system::Config> WeightInfo for HydraWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(1 as u64))
 			.saturating_add(T::DbWeight::get().writes(1 as u64))
 	}
+	// Storage: XcmpQueue QueueConfig (r:1 w:0)
+	// Proof Skipped: XcmpQueue QueueConfig (max_values: Some(1), max_size: None, mode: Measured)
+	// Storage: XcmpQueue DeferredXcmMessages (r:1 w:1)
+	// Proof Skipped: XcmpQueue DeferredXcmMessages (max_values: None, max_size: None, mode: Measured)
+	// Storage: XcmpQueue CounterForOverweight (r:1 w:1)
+	// Proof: XcmpQueue CounterForOverweight (max_values: Some(1), max_size: Some(4), added: 499, mode: MaxEncodedLen)
+	// Storage: XcmpQueue OverweightCount (r:1 w:1)
+	// Proof Skipped: XcmpQueue OverweightCount (max_values: Some(1), max_size: None, mode: Measured)
+	// Storage: XcmpQueue Overweight (r:100 w:100)
+	// Proof Skipped: XcmpQueue Overweight (max_values: None, max_size: None, mode: Measured)
+	fn service_deferred() -> Weight {
```

Comment:
It is quite heavy as it is hard to bound, so we assume a pretty bad worst case.
We decided it's not worth it to optimize as it will be executed by a privileged origin (the `ExecuteDeferredOrigin`).
It is this extrinsic:
https://github.com/galacticcouncil/cumulus/pull/3/files#diff-3d3febd8cfcd42ba5bb949d63fdafb9a71da3db1feb35a45601404b64fcd70c4R331